.env

```diff
@@ -13,3 +13,4 @@ WORDPRESS_LOGIN=admin
 WORDPRESS_PASSWORD=password
 WORDPRESS_EMAIL=admin@woocommercecoree2etestsuite.com
 WP_VERSION=5.4
```

Comment:
Should `WP_VERSION` be updated here? We no longer support WP 5.4.

Reply:
This is one of the unused parts I mentioned here:

> the docker configuration we're currently using seems to have a lot of unused parts and could likely be cleaned up a bit. What I added here complicates things a bit more.

The actual test suite currently ignores this and always uses the latest WP.